recipes-bsp/tegra-binaries/tegra186-flashtools-native_32.1.0.bb

```diff
@@ -7,6 +7,20 @@ SSTATE_SWSPEC = "sstate:tegra-binaries-native::${PV}:${PR}::${SSTATE_VERSION}:"
 STAMP = "${STAMPS_DIR}/work-shared/L4T-native-${SOC_FAMILY}-${PV}-${PR}"
 STAMPCLEAN = "${STAMPS_DIR}/work-shared/L4T-native-${SOC_FAMILY}-${PV}-*"
 
+SRC_URI += "file://nvargus-daemon.init \
+           file://nvargus-daemon.service \
+           file://nvpmodel.init \
+           file://nvpmodel.service \
+           file://nvphs.init \
+           file://nvphs.service \
+           file://nvs-service.init \
+           file://nvs-service.service \
+           file://nvstartup.init \
+           file://nvstartup.service \
+           file://tegra186-flash-helper.sh \
+           file://tegra194-flash-helper.sh \
+           file://tegra210-flash-helper.sh \
+           "
```

Comment:
Why are you adding these files to this recipe?

Reply:
Oh because I saw you added those files to this recipe,  recipes-bsp/tegra-binaries/tegra186-flashtools-native_32.2.1.bb

in this commit https://github.com/madisongh/meta-tegra/commit/c45dc8bae40f533aa7cfd062f3b5d86069f85259

Reply:
Hmm, now I have to ask myself why I did that. :)

Reply:
If I remember correct, I think this create the `STAGING_BINDIR_NATIVE` variable and use in this line, but not very sure :):

in image_types_tegra.bbclass
```
cp -R ${STAGING_BINDIR_NATIVE}/tegra186-flash/* .
```

Reply:
Yeah, I think this was an oversight on my part... I think I was using `devtool`, which snuck in the duplicate entries on me.  No real harm in keeping them here to match, and I have a bigger overhaul of these recipes in the works coming soon to master anyway, so this is fine as-is.